bin/smoke-test.sh

```diff
@@ -592,13 +620,14 @@ while getopts "h?t:v:p:r:u:s:fxlLCcd2" opt; do
             echo "  -l         Disable location-trusted work-around"
             echo "  -L         Enable location-trusted work-around"
             echo "  -2         Force client not to use TLS v1.3 as work-around"
-            echo "  -t <token> Use <token> for non-X.509 authn to target"
+            echo "  -t <token> Use <token> for non-X.509 authn to target (valid values: macaroon, scitoken, wlcg)"
             echo "  -d         Include additional logging"
             echo "  -v <vo>    Test as member of VO <vo>"
             echo "  -p <url>   Use <url> as the public link for TPC PULL"
             echo "  -r <url>   Use <url> as the VO-private link for TPC PULL"
             echo "  -u <url>   Use <url> as the base for HTTP PUSH"
             echo "  -s <url>   SciToken server"
+            echo "  -o <name>  oidc-agent account name (for use with wlcg tokens)"
             echo
             echo "Defaults:"
             echo "  -p $THIRDPARTY_UNAUTHENTICATED_URL"
```

Comment:
[github doesn't let me add this comment any further down in the file]

As above, if you add support for default WLCG VO values, please also document them here.